### PR TITLE
Localized sidebar and appView

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: common/i18n/i18nFactory.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const i18next = require('i18next')
+
+const enTranslation = require('./resources/en')
+
+const createParams = lang => ({
+  fallbackLng: 'en',
+  debug: true,
```
Comment (minotogna) [inline common/i18n/i18nFactory.js]: Why debug true?

File: webapp/loggedin/appSideBar/appSideBarFooter.js [hunk 2]
```diff
@@ -1,31 +1,42 @@
...
-          ? 'Open Foris' : 'OF'
-      }
-    </a>
-  </div>
-)
+class AppSideBarFooter extends React.PureComponent {
+  render () {
+    const { logout, opened } = this.props
+    const { t } = this.context.i18n
+
```
Comment (minotogna) [inline webapp/loggedin/appSideBar/appSideBarFooter.js]: It's not clear when reading just t in the code.
What if we import the ```const i18n = this.context.i18n``` and then use it as```i18n.t('key')```?

File: webapp/loggedin/appView.js [hunk 3]
```diff
@@ -1,28 +1,46 @@
...
+
+      </div>
+    )
+  }
+}
 
-    </div>
-  )
+const mapStateToProps = (state) => {
+  return { i18n: AppState.getState(state).i18n }
```
Comment (minotogna) [inline webapp/loggedin/appView.js]: Let's create a method in AppState.getI18n?

File: webapp/loggedin/modules/home/homeView.js [hunk 4]
```diff
@@ -43,6 +45,7 @@ class HomeView extends React.Component {
 
   render () {
     const { surveyInfo } = this.props
+    const { t } = this.context.i18n
```
Comment (minotogna) [inline webapp/loggedin/modules/home/homeView.js]: As above

File: common/i18n/i18nFactory.js (hunk 1 above)
Comment (sgiaccio) [reply common/i18n/i18nFactory.js]: Just for the time being, for development... but right should be removed on master

File: webapp/loggedin/appSideBar/appSideBarFooter.js (hunk 2 above)
Comment (sgiaccio) [reply webapp/loggedin/appSideBar/appSideBarFooter.js]: ok

File: webapp/loggedin/modules/home/homeView.js (hunk 4 above)
Comment (sgiaccio) [reply webapp/loggedin/modules/home/homeView.js]: ok

File: webapp/loggedin/appView.js (hunk 3 above)
Comment (sgiaccio) [reply webapp/loggedin/appView.js]: ok

File: common/i18n/i18nFactory.js (hunk 1 above)
Comment (minotogna) [reply common/i18n/i18nFactory.js]: you could use https://github.com/openforis/arena/blob/481d40c0da07e4dc1327761f859cb6398562c318/common/processUtils.js#L5

File: webapp/survey/surveyInfo/reducer.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { exportReducer } from '../../utils/reduxUtils'
 
 import Survey from '../../../common/survey/survey'
 
-import { assocSurveyInfoProp, assocSurveyInfoValidation, markDraft, markPublished } from './surveyInfoState'
+import { assocSurveyInfoProp, assocSurveyInfoValidation, markDraft } from './surveyInfoState'
```
[the remark this answers is not in the document]
Comment (sgiaccio) [reply webapp/survey/surveyInfo/reducer.js]: ok

File: common/i18n/i18nFactory.js (hunk 1 above)
Comment (sgiaccio) [reply common/i18n/i18nFactory.js]: ok, done

**Pull request merged**